src/common/src/FileUpload/hooks.ts

```diff
@@ -11,42 +11,61 @@ export interface CustomFile extends File {
 export type CustomFileOrRejection = CustomFile & FileRejection;
 export type Files = CustomFileOrRejection[];
 
+let tmpId = 0;
```

Comment:
Ik weet niet of ik dit heel mooi vindt het wordt dan een application wide per client forever increasing value.
Ik zou dit zelf middels een useRef hook in deze hook implementeren met een .current value gebruik.

Daarnaast denk ik dat dit een hook zou moeten zijn waarvoor unit tests zouden moeten bestaan.
Nu hebben we een bug, en heb je snel een bugfix gemaakt, maar hoe zorgen we ervoor dat er in de toekomst niet weer een bug voorkomt.

Reply:
I agree. Much in this hook wasn't covered to begin with. I will make a story to fix this. For now this is blocking to reach a sprint goal.
Can you give me a pointer how a ref would solve this? I tried useState earlier, but there's race conditions between state having propagated and the next increment being required. These id's are only for internal tracking and have no meaning beyond that for the record.